src/features/collapsible-sidebar/CollapsibleSidebar.scss

```diff
@@ -111,16 +111,9 @@ $left-sidebar-z-index: 40 !default;
     }
 }
 
-@include breakpoint($small-screen) {
-    .Body.is-responsive-web {
-        .bdl-CollapsibleSidebar.is-expanded {
-            width: 100vw;
-
-            .bdl-CollapsibleSidebar-nav ul {
-                // using fixed width due to performance issues during expand transition
-                width: calc(100vw - (#{$collapsible-side-bar-h-padding} * 2));
-            }
-        }
+@include large-size {
```

Comment:
I sorted media queries by width precedence since our current method of targeting sizes uses max-width instead of expact widths like 200-400. It's my opinion that we should correct how widths are targeted in the media-queries file so that we don't have tons of queries in the document and need to worry about order precedence

Reply:
Are you're saying that if we have multiple active media queries (due to a mix of old breakpoints and new ones) whichever is smaller should win? If that is the case, I don't know the answer to that---whether the precedence should be based on size, or whether the newer stuff should always override, or whether it would have to be a case-by-case thing.

Reply:
Ah, do the old ones use `max-width`?

Reply:
The old `large-size` looks like this:
```
@mixin large-size {
    @media (min-width: #{$medium-width + 1px}) {
        @content;
    }
}
```
Seems like a very confusing situation with some old media queries using `min-width` and new ones using `max-width`.

Reply:
In these cases, it will take by order (whoever comes last)

Reply:
I am also thinking about non-overlapping styles. On the one hand, sometimes you may want styles to "cascade" inclusively across breakpoints (so like if large specifies block and blue, and small specifies red, then small turns out block and red). This can get pretty confusing, imo, when you have both min-width and max-width, styles bubbling up and down, as it were. The fact that you can't inherit styles from other media queries might be seen as a drawback to exact media queries---but personally, I would rather explicitly set the styles twice than be super confused about what is getting inherited from where.